tests/schemas/block.schema.js

```diff
@@ -36,6 +36,29 @@ const blockSchema = {
 	isFinal: Joi.boolean().optional(),
 };
 
+const blockSchemaVersion5 = {
+	id: Joi.string().required(),
+	height: Joi.number().integer().min(1).required(),
+	version: Joi.number().required(),
+	timestamp: Joi.number().integer().min(1).required(),
+	generatorAddress: Joi.string().optional(),
+	generatorPublicKey: Joi.string().required(),
+	generatorUsername: Joi.string().optional(),
+	transactionRoot: Joi.string().required(),
+	// previousBlockId: Joi.string().required(),
```

Comment:
This should be enabled.
```suggestion
	previousBlockId: Joi.string().required(),
```

Reply:
We are not getting `previousBlockId` in a block response. It need to be resolved and enabled later.